src/api/index.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+
+import { API_KEY, BASE_API_URL } from '../constants';
+
+const UNITS = 'metric';
+const FORECAST_QUERY_URL = `${BASE_API_URL}/forecast`;
+
+export const get5DaysForecastByCoords = async (location) => {
```

Comment:
You can deconstruct location here, this way I would also know expectation about parameters.